mlos_bench/mlos_bench/environments/remote/remote_env.py

```diff
@@ -151,6 +156,19 @@ def teardown(self) -> None:
             _LOG.info("Remote teardown complete: %s :: %s", self, status)
         super().teardown()
 
+    def _get_env_params(self) -> Dict[str, str]:
+        """
+        Get the *shell* environment parameters to be passed to the remote script.
+
+        Returns
+        -------
+        env_params : Dict[str, str]
+            Parameters to pass as *shell* environment variables into the script.
+            This is usually a subset of `_params` with some possible conversions.
+        """
+        keys = self._params.keys() if self._script_params is None else self._script_params
+        return {re.sub(r"\W", "_", key): str(self._params[key]) for key in keys}
```

Comment:
That might be too broad.  Maybe make it more explicit the type of characters we do or do not support?

Reply:
done